fetch_rapids.cmake

```diff
@@ -12,6 +12,8 @@
 # the License.
 # =============================================================================
 if(NOT EXISTS ${CMAKE_CURRENT_BINARY_DIR}/RMM_RAPIDS.cmake)
+  set(rapids-cmake-repo bdice/rapids-cmake)
```

Comment:
Move these 2 lines outside the `if` so that we get this behavior even if `RMM_RAPIDS.cmake` exists. Might not change CI output, but helps when testing locally.